src/components/onboarding/screens/create.tsx

```diff
@@ -49,6 +48,32 @@ const createTimeoutLoop = (setState: (item: MockData) => void, arr: MockData[],
     }, (index + 1) * 2400)
   );
 
+const explainerData: ExplainerCardProps[] = [
+  {
+    title: 'Private data storage',
+    imageUrl: '/assets/images/icon-delay-private.svg',
+  },
+  {
+    title: 'Always-on encryption',
+    imageUrl: '/assets/images/icon-delay-padlock.svg',
+  },
+  {
+    title: 'Access to 100s of apps',
+    imageUrl: '/assets/images/icon-delay-apps.svg',
+  },
+  {
+    title: 'This will not display',
+    imageUrl: '',
+  },
+];
+
+const preloadImages = (images: { imageUrl: string }[]) => {
```

Comment:
very cool technique!

Reply:
Could've inlined them as components, but this does just as good a job imo